client/src/main/scala/drt/client/components/TerminalComponent.scala

```diff
@@ -177,7 +176,7 @@ object TerminalComponent {
         }))
       })
     })
-    .componentDidMount((p) => Callback.log("TerminalComponent did mount"))
+    .componentDidMount(_ => Callback.log("TerminalComponent did mount"))
```

Comment:
not sure we even need this log anymore to be honest.

Reply:
true i'll remove it. i was just doing blind intellij-suggested tidy ups